keras/utils/multi_gpu_utils.py

```diff
@@ -46,12 +47,18 @@ def multi_gpu_model(model, gpus=None):
             (see usage example below).
         gpus: Integer >= 2 or list of integers, number of GPUs or
             list of GPU IDs on which to create model replicas.
+        cpu_merge: A boolean value to identify whether to force
+            merging model weights under the scope of CPU or not.
+        cpu_relocation: A boolean value to identify whether to
+            replias the model under the scope of CPU. If the model
```

Comment:
typo: "create the model's weights under..."

also prefer using "the CPU" since we always assume one CPU